plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/TestJdbcRecordSetProvider.java

```diff
@@ -64,7 +64,7 @@
     private JdbcColumnHandle textShortColumn;
     private JdbcColumnHandle valueColumn;
 
-    private ExecutorService executor;
+    private ListeningExecutorService executor;
 
     @BeforeClass
     public void setUp()
```

Comment:
Please add a test like `testExplainAnalyzePhysicalReadWallTime` to `BaseJdbcConnectorTest`